cicd/pipelines/creating-applications-with-cicd-pipelines.adoc

```diff
@@ -63,13 +63,20 @@ include::modules/op-creating-webhooks.adoc[leveloffset=+1]
 
 include::modules/op-triggering-a-pipelinerun.adoc[leveloffset=+1]
 
+include::modules/op-enabling-monitoring-of-event-listeners-for-triggers-for-user-defined-projects.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
```

Comment:
From the checklist: “Additional resources” and “Next steps” titles use unique IDs."
How about:
```suggestion
[role="_additional-resources_op-enabling-monitoring-of-event-listeners-for-triggers-for-user-defined-projects"]
```